deployment/initialize_keycloak.sh

```diff
@@ -30,14 +30,15 @@ docker logs "$keycloak_initializer_container_name"
 if ls "$keycloak_user_dir"/*-users-*.json &>/dev/null; then
   echo "Testing if the number of current users matches the number of exported users"
   current_users=$(sudo docker exec $keycloak_database_container_name psql -U keycloak -d keycloak -t -c "select count(*) from user_entity where realm_id = 'datalandsecurity'")
-  current_technical_users=$(sudo docker exec $keycloak_database_container_name psql -U keycloak -d keycloak -t -c "select count(*) from user_entity where realm_id = 'datalandsecurity' and username in ('data_reader','data_uploader','data_reviewer','data_admin')")
+  current_technical_users=$(sudo docker exec $keycloak_database_container_name psql -U keycloak -d keycloak -t -c "select count(*) from user_entity where realm_id = 'datalandsecurity' and username in ('data_reader','data_uploader','data_reviewer','data_admin','service-account-dataland-batch-manager')")
   all_users=$(sudo docker exec "$keycloak_initializer_container_name" bash -c 'grep -l username /keycloak_users/datalandsecurity-users-*.json | wc -l')
   technical_users=$(sudo docker exec --env USER_PATTERN='"username" : "data_(reader|uploader|reviewer|admin)"' "$keycloak_initializer_container_name" bash -c 'grep -E -l "$USER_PATTERN" /keycloak_users/datalandsecurity-users-*.json | wc -l')
   test_users=$(sudo docker exec "$keycloak_initializer_container_name" bash -c 'grep -E -l \"test_user.*@dataland.com\" /keycloak_users/datalandsecurity-users-*.json | wc -l')
+  service_users=$(sudo docker exec --env USER_PATTERN='"username" : "service-account-dataland-batch-manager"' "$keycloak_initializer_container_name" bash -c 'grep -E -l "$USER_PATTERN" /keycloak_users/datalandsecurity-users-*.json | wc -l')
   actual_users=$((current_users-current_technical_users))
-  expected_users=$((all_users-test_users-technical_users))
+  expected_users=$((all_users-test_users-technical_users-service_users))
   echo "The new instance contains a total of $current_users users with $current_technical_users technical users (Actual users: $actual_users)"
-  echo "The old instance contained a total of $all_users users with $technical_users technical users and $test_users test users (Actual users: $expected_users)"
+  echo "The old instance contained a total of $all_users users with $technical_users technical users and $test_users test users and $service_users service users (Actual users: $expected_users)"
```

Comment:
Ich habe das gerade zum ersten mal komplett durchgelesen, und ich würde vorschlagen das etwas leserlicher zu machen.
Lass uns das aber in einem Call kurz besprechen.

Reply:
Haben wir erledigt am Telefon.